pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -141,11 +142,26 @@ func (o *ResourcesOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args
 		return err
 	}
 
+	var includeUninitialized bool
+	if o.All {
+		// include the uninitialized objects by default
+		// unless explicitly set --include-uninitialized=false
+		includeUninitialized = true
+	}
+	if o.Selector != "" {
```

Comment:
is `Selector != ""` when I do `kubectl set pods/foo ...`? Trying to understand what this clause is for.